ietf/utils/test_utils.py

```diff
@@ -258,7 +258,7 @@ def assertMailboxContains(self, mailbox, subject=None, text=None, count=None):
             assert isinstance(text, str)
             mlist = [ m for m in mlist if text in get_payload_text(m) ]
         if count and len(mlist) != count:
-            sys.stderr.write("Wrong count in assertMailboxContains().  The complete mailbox contains %s emails:\n\n" % len(mailbox))
+            sys.stderr.write("Wrong count in assertMailboxContains().  The complete mailbox contains %s emails, only %s out of them are valid:\n\n" % (len(mailbox), len(mlist)))
```

Comment:
by `are valid`, do you mean `match the assert criteria`?

Reply:
No, I mean that they contain the right string, i.e., the subject line is correct.

Should I modify the text  to make it clear ?

Reply:
yes please

Reply:
Marking this as changes requested.